src/main/java/org/broadinstitute/hellbender/engine/AbstractConcordanceWalker.java

```diff
@@ -162,6 +213,59 @@ private static VCFHeader getHeader(final FeatureDataSource<VariantContext> sourc
         return (VCFHeader)header;
     }
 
+    protected ConcordanceState getConcordanceState(final Genotype truth, final Genotype eval, final boolean evalWasFiltered) {
```

Comment:
Is this introducing a change where concordance is based on each sample's FORMAT field rather than the variant as a whole?  Also, does the agreement of genotypes here ultimately come from the 0/0, 0/1 etc of the GT attribute?

If so, we will need to keep the old behavior as an option for the somatic pipeline.

Reply:
You are correct, this changes the behavior in the abstract class to be genotype level concordance, so coming from the GT attribute.

For tools like Concordance which already existed, the old behavior is maintained through the additional abstract layer of the `SingleSampleConcordanceWalker` class, which tests for concordance on a site level (as opposed to genotype level) based on having the same ref and first alt alleles.  Though I think that name is a bit confusing, so I will change it to something a little clearer (I'm think  AlleleConcordanceWalker?)